### DISCUSSION TITLE
Milestone 2 Preparation

### DISCUSSION THREAD

**Discussion by Tubagc** [bounswe/bounswe2020group6#252] [Ideas]
We will carry out our planned tasks on the [Project Plan](https://github.com/bounswe/bounswe2020group6/blob/master/Files/Templates/Project%20Plan1.pdf). The dates given may vary. 

For now, it is decided that the team carry out tasks up to 50, 

### 🔮 The back-end team:
- [ ]  inviting a user, accepting and rejecting requests
- [ ]  seeing , accepting, and rejecting requests
- [ ]  status of papers/projects
- [ ]  home page
- [ ]  google scholar linking
- [ ]  simultaneous document preparation
- [ ] filtering system 
- [ ] follow functionality

### 🌺 The front-end  team should implement: 
- [ ]  inviting a user, accepting and rejecting requests
- [ ]  seeing , accepting, and rejecting requests
- [x]  Logout
- [ ]  status of papers/projects
- [ ]  home page
- [ ]  google scholar linking
- [ ]  simultaneous document preparation
- [ ] filtering system 
- [ ] follow functionality

### 🤖 The android  team should implement: 
- [ ]  inviting a user, accepting and rejecting requests
- [ ]  seeing , accepting, and rejecting requests
- [x]  Logout
- [ ]  status of papers/projects
- [ ]  home page
- [ ]  google scholar linking
- [ ]  simultaneous document preparation
- [x] filtering system 
- [ ] follow functionality

**Comment by emilcan-arican** [comment]
As frontend team, we believe, the term "filtering system" is not very clear. Can we clarify what is expected to be achieved?
